reward/reward_config_cache_test.go

```diff
@@ -349,7 +348,7 @@ func TestRewardConfigCache_get_exist(t *testing.T) {
 	rewardConfigCache := newRewardConfigCache(testGovernance)
 
 	for i := 0; i < len(testCases); i++ {
-		testGovernance.setTestGovernance(testCases[i].epoch, testCases[i].mintingAmount, testCases[i].ratio, testCases[i].unitprice, testCases[i].useGiniCoeff, testCases[i].deferredTxFee)
+		testGovernance.setTestGovernance(testCases[i].epoch, testCases[i].mintingAmount, testCases[i].ratio, testCases[i].useGiniCoeff, testCases[i].deferredTxFee)
```

Comment:
Is the unitprice parameter missing from governance after the hard fork?
If so, I think it would be nice to have a test case to check if the parameter is missing after the hard fork.

Reply:
@hqjang-pepper I don't think the parameter itself will be removed. The txpool will be modified to simply ignore UnitPrice and respect calculateBaseFee instead. Refer to [#1382](https://github.com/klaytn/klaytn/pull/1382/files#diff-4a20f160f5182ec35d76a3566352906f577b84fbf3b242b922c4d2fc9c219cd7R422).

@kjeom Can we keep the rewardConfigCache.unitPrice? unitPrice still has to function because `getTotalTxFee` refers to unitPrice before hardfork. Also we don't have to remove the unitPrice. `getTotalTxFee` ignoring the unitPrice after hardfork would be sufficient.

Reply:
@blukat29 Thanks for answering this topic.
I also think we need `rewardConfigCache.unitPrice` for blocks before the hard fork